src/PrestaShopBundle/Resources/config/services/bundle/services.yml

```
@@ -53,6 +53,8 @@ services:
 
     prestashop.service.translation:
         class: PrestaShopBundle\Service\TranslationService
+        arguments:
+            - "%default_theme%"
```

```suggestion
            - env(PS_THEME_NAME)
```
Could be better wdyt?